src/test/java/com/volasoftware/tinder/controllers/RegistrationControllerIntegrationTests.java

```diff
@@ -0,0 +1,80 @@
+package com.volasoftware.tinder.controllers;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.volasoftware.tinder.constants.Gender;
+import com.volasoftware.tinder.dtos.AccountDTO;
+import com.volasoftware.tinder.dtos.RegisterDTO;
+import com.volasoftware.tinder.services.contracts.AccountService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+@WebMvcTest(RegisterController.class)
+public class RegistrationControllerIntegrationTests {
+
+  @Autowired
+  private RegisterController registerController;
+
+  @Autowired
+  private MockMvc mockMvc;
+
+  @MockBean
+  private AccountService accountService;
+
+  @Test
+  void contextLoads() {
```

Comment:
No need of this test. If context is not load, it shows an error

Reply:
removed